src/config/routes.rb

```diff
@@ -33,4 +33,10 @@
       end
     end
   end
+
+  # Authentication Routes
+  match 'auth/:provider/callback', to: 'sessions#create', via: [:get, :post]
```

Comment:
- [] Remove use of `match
- [] Just POST to `sessions#create` -> somehow there will be error from omniauth during callback if I remove the `GET` method from `auth/:provider/callback`